mainwindow.ui

```diff
@@ -1,110 +1,111 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
- <class>MainWindow</class>
```

Comment:
Why is this massive re-format necessary?... it creates a lot of churn and makes it harder to review the actual changes. Also, if a change like this must be done, I think it should at least be broken into a separate commit. Also also, this file was created and is typically edited using the QT interface designer tool.